client/src/pages/ProfileSetting/ProfileSetting.tsx

```diff
@@ -1,18 +1,37 @@
+import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useHistory } from 'react-router-dom';
 import ProfilePhoto from '../../components/ProfilePhoto/ProfilePhoto';
+import getProfile from '../../helpers/APICalls/getProfile';
+import { useSnackBar } from '../../context/useSnackbarContext';
 
 export default function ProfileSetting(): JSX.Element {
   const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [profile, setProfile] = useState({});
 
   const { loggedInUser } = useAuth();
 
   const history = useHistory();
 
+  useEffect(() => {
```

Comment:
Note: Just keep in mind that we may not need this once Mayank merges his PR as his branch fetches the profile when the user logs in